release-notes/7.0/preview/api-diff/preview1/Microsoft.AspNetCore.App/7.0-preview1_Microsoft.Extensions.Configuration.Ini.md

```diff
@@ -0,0 +1,11 @@
+# Microsoft.Extensions.Configuration.Ini
+
+``` diff
+ namespace Microsoft.Extensions.Configuration.Ini {
+     public class IniStreamConfigurationProvider : StreamConfigurationProvider {
+-        public static IDictionary<string, string> Read(Stream stream);
++        public static IDictionary<string, string?> Read(Stream stream);
```

Comment:
@eerhardt, why does this return `IDictionary<string, string?>` rather than `IDictionary<string, string>`?

Reply:
https://github.com/dotnet/runtime/pull/60368/files#r783288134